brambling/tests/functional/test_invites.py

```diff
@@ -19,7 +19,13 @@ def test_subject__event_editor(self):
         invite.send(Site('test.com', 'test.com'), content=invite.get_content())
         self.assertEqual(len(mail.outbox), 1)
         self.assertEqual(mail.outbox[0].subject, "{} has invited you to edit {}".format(invite.user.get_full_name(), event.name))
-
+    def test_subject__event_editor_apostrophe(self):
+	event= EventFactory(name="James's Test Event")
```

Comment:
The indentation here is messed up. Looks like the indentation inside this function (and all the test functions in this PR) is using tabs instead of spaces. One thing that might help is installing the EditorConfig plugin for sublime text – then it'll automatically make sure your code conforms to our style guide.

I'm actually a little confused as to how these tests are passing; usually python complains about mismatched indenting...